src/lib.rs

```diff
@@ -132,23 +132,15 @@
 //! * structure!() macro takes a literal string as an argument.
 //! * It's called `structure` because `struct` is a reserved keyword in Rust.
 
-#[macro_use]
-extern crate proc_macro_hack;
 
```

Comment:
A `#![cfg_attr(not(feature = "std"), no_std)]` is needed here.

Reply:
Fixed in 696391c.